static/js/app.ts

```diff
@@ -853,20 +871,11 @@ export function runPythonProgram(this: any, code: string, sourceMap: any, hasTur
     }
   }
 
-  //Saving the variable button because sk will overwrite the output div
-  const variableButton = outputDiv.find('#variable_button');
-  const variables = outputDiv.find('#variables');
-  outputDiv.empty();
-  outputDiv.append(variableButton);
-  outputDiv.append(variables);
-
-  const storage = window.localStorage;
   let skulptExternalLibraries:{[index: string]:any} = {
       './extensions.js': {
         path: theStaticRoot + "/vendor/skulpt-stdlib-extensions.js",
```

Comment:
In light of our new changes, might it be safer to load all Skuplt recourses ourselves? I worry a bit that they might update stuff that breaks our new code (but I can't oversee whether that is a reasonable worry, what do you think @jpelay?)

We added this CDN link in https://github.com/hedyorg/hedy/pull/4490 for performance reasons (so we will make @rix0rrr sad haha)

Reply:
Sorry, Felienne, but I don't fully understand what you mean by load all Skulpt resources ourselves? I didn't remove that part of the code, the `storage = window.localStorage` was being used to know if we were in debug mode, no need to do that since now we have two clearly differentiated execution paths!

Reply:
Ah never mind, my bad! I thought thestaticroot was referring to a generic CDN hosting libraries (something like https://developers.google.com/speed/libraries) but I see now it just refers to our own repo, so we can leave it as is!